Makefile

```diff
@@ -430,7 +434,10 @@ ifeq ($(RUN_MODE),kubernetes)
 	kubectl -n $(KUBE_NAMESPACE) scale deploy/workflow-controller --replicas 1
 	kubectl -n $(KUBE_NAMESPACE) scale deploy/argo-server --replicas 1
 endif
-	kubectl -n $(KUBE_NAMESPACE) wait --for=condition=Ready pod --all -l app --timeout 2m
+ifeq ($(PROFILE),prometheus)
+	kubectl -n $(KUBE_NAMESPACE) scale deploy/prometheus --replicas 1
+endif
+	kubectl -n $(KUBE_NAMESPACE) wait --for=condition=Ready pod --all -l app --timeout 1m || true
```

Comment:
In my experience `kubectl wait` hangs/gets stuck even after all pods are ready (which happens after around 30 seconds). Confirmed this by running the same command in another terminal window while it is stuck, and in that case the command completes successfully.

We still want to have a wait before we do `port-forwarding`, but we don't want this flakey command to kill start up, so I lowered the timeout and added ` || true` to ensure this flakey command doesn't stop the build. If the pods actually aren't ready, the next command will fail anyway (which in my experience does not happen).

Reply:
I've noted this is flakey too. Could we make the wait specific? E.g. 

```
kubectl -n $(KUBE_NAMESPACE) wait --for=condition=Ready pod -l app=prometheus
```

Reply:
might need a short sleep before hand

Reply:
Added sleep